frontend/src/locales/en.yml

```diff
@@ -556,6 +556,8 @@ quick-start:
   visit-database: '@:common.visit @:common.databases'
   visit-environment: '@:common.visit @:common.environments'
   add-a-member: Add a member
+  use-kbar: Use kbar
```

Comment:
Use kbar (press cmd+k) ?

Reply:
Or maybe

Use kbar (cmd-k)

Reply:
so maybe we should remove the tooltip then

Reply:
I think the tooltips is good as well. It's more readable.

My suggestion is intended to cram extra text in that limited space.

Reply:
I prefer "Use kbar (cmd-k)" without a tooltip due to the duplicating of their contents.